uberfire-security/uberfire-security-management/uberfire-security-management-api/src/main/java/org/uberfire/ext/security/management/api/UsersManager.java

```diff
@@ -0,0 +1,82 @@
+package org.uberfire.ext.security.management.api;
+
+import org.jboss.errai.security.shared.api.identity.User;
+import org.uberfire.ext.security.management.api.exception.GenericUsersSystemManagementException;
+
+import java.util.Collection;
+
+/**
+ * <p>The Users Manager.</p>
+ * <p>The user manager provide additional methods to implement from the entity manager ones.</p>
+ * 
+ * @since 0.8.0
+ */
+public interface UsersManager extends AbstractEntityManager<User> {
```

Comment:
Just a comment that I'd prefer singular UserManager instead of UsersManager but disregard if you prefer this.